<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Port_From_Button">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>20</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse Files</string>
    </property>
   </widget>
   <widget class="QLabel" name="Port_From_Label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>531</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Port From:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Port_To_Label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>531</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Port To:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Port_To_Button">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>70</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse Files</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Sheet_Name_Text">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>541</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Sheet_Name_Label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>541</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>New Sheet Name:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Submit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>187</width>
      <height>57</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Quit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>200</y>
      <width>187</width>
      <height>57</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
